dll/win32/shell32/folders/CDrivesFolder.cpp

```diff
@@ -589,6 +590,13 @@ HRESULT WINAPI CDrivesFolder::ParseDisplayName(HWND hwndOwner, LPBC pbc, LPOLEST
     if (PathGetDriveNumberW(lpszDisplayName) < 0)
         return E_INVALIDARG;
 
+    /* check if this drive actually exists */
+    if (!GetVolumePathNameW(lpszDisplayName, volumePathName, MAX_PATH) ||
```

Comment:
Since you are using a buffer now, it would make sense to use `_countof(volumePathName)` or `ARRAYSIZE` (depending what works in this binary.

Reply:
`_countof` did the trick!  I've pushed new code